.mergify.yml

```diff
@@ -121,11 +91,12 @@ pull_request_rules:
       - 'check-success=encryption-pvc-db-wal'
       - 'check-success=encryption-pvc-kms-vault-token-auth'
       - 'check-success=TestCephSmokeSuite (v1.16.15)'
```

Comment:
```suggestion
      - 'check-success=TestCephSmokeSuite (v1.15.12)'
```

Reply:
No, that's how the CI is setup, so I'm not changing it.

Reply:
okay :+1: